wandb/apis/internal.py

```diff
@@ -137,6 +137,18 @@ def disabled(self):
         except configparser.Error:
             return False
 
+    def sync_spell(self, url, env=None):
+        """Syncs this run with spell"""
+        try:
+            env = env or os.environ
+            wandb.config.update({"spell_url": env.get("SPELL_RUN_URL")})
+            return requests.put(env.get("SPELL_API_URL", "https://api.spell.run/wandb_url"), json={
+                "access_token": env.get("WANDB_ACCESS_TOKEN"),
```

Comment:
Ideally there are a few things we could do here to prevent misuse.
1) Generate/enforce certificates for consumers of this request.
2) Create service access tokens which operate on a users behalf
3) Require https for SPELL_API_URL

This seems good enough for an initial integration, but we can consider locking things down a bit more in the future.
This seems fine